kubernetes/check.py

```diff
@@ -34,6 +34,8 @@
     'cpu.*.total']
 DEFAULT_COLLECT_EVENTS = False
 DEFAULT_NAMESPACES = ['default']
+DEFAULT_SEND_CPU_AS_CORES = False
```

Comment:
Oh good catch! originally this was configurable - I had the mistaken notion that the metrics we being sent in "millicores", which an option to use in the kube apis (instead of just cores). Since some folks might have been expecting the same unit they set in their configs, I made it toggle-able. 

Since that's _not_ what's happening (it's actually nanocores, heh) and this requires a change to `metadata.csv` (which isn't reasonably toggle-able, I think?) I made it non-optional.

Would just removing the constant make sense? I'm not sure there's a good way to make it actually changeable